test/conformance/CMakeLists.txt

```diff
@@ -38,6 +41,10 @@ function(add_test_adapter name adapter)
     endif()
 
     set(TEST_ENV UR_ADAPTERS_FORCE_LOAD="$<TARGET_FILE:ur_${adapter}>")
+    if("${ARGN}" STREQUAL "v2")
+        list(APPEND TEST_ENV "UR_L0_USE_QUEUE_V2=1")
+        list(APPEND TEST_ENV "UR_L0_USE_IMMEDIATE_COMMANDLISTS=1")
```

Comment:
why?

Reply:
Setting UR_L0_USE_IMMEDIATE_COMMANDLISTS is needed to make sure that the dispatcher will create v2_queue (not legacy queue). Alternatively we can modify the dispatcher to always use immediate cmd lists whenever UR_L0_USE_QUEUE_V2=1 is set.

But eventually, we will probably have separate test cases for immediate = 0 (once this path is implemented) anyway.